### PR TITLE
added documentation for `aria-valuemax` `aria-valuemin` `aria-valuenow` `aria-valuetext`

### PR DESCRIPTION
**Pull request by dakshbhardwaj**
This PR adds documentation for `aria-valuemax`, `aria-valuemin`, `aria-valuenow`,`aria-valuetext` props.

Related to commit https://github.com/facebook/react-native/commit/e8739e962de3398bc7e42675b1d87ab35993f705

and issue https://github.com/facebook/react-native/issues/34424

### CODE REVIEW

File: docs/touchablewithoutfeedback.md [hunk 1]
```diff
@@ -219,6 +219,46 @@ See the [Accessibility guide](accessibility.md#accessibilityvalue-ios-android) f
 
 ---
 
+### `aria-valuemax`
+
+It Represents the maximum value for range-based components, such as sliders and progress bars. Has precedence over the `accessibilityValue.max` prop.
```
Comment (Simek) [inline docs/touchablewithoutfeedback.md:224]:
```suggestion
Represents the maximum value for range-based components, such as sliders and progress bars. Has precedence over the `max` value in the `accessibilityValue` prop.
```

Please remove "It" at the beginning of each description, ensure that case is correct and rephrase the precedence sentence as seen above in other entries added.

**Review by Simek — CHANGES_REQUESTED** [review]
See the comment.

**Comment by dakshbhardwaj** [comment]
> See the comment.

Thanks for the review @Simek. i have pushed down the changes

**Comment by Simek** [comment]
Can you resolve the conflicts created after the merge of other PRs?

**Comment by dakshbhardwaj** [comment]
> Can you resolve the conflicts created after the merge of other PRs?

@Simek i have resolved the conflicts

**Review by Simek — APPROVED** [review]
LGTM, thank you for those additions! 🙏

**Pull request merged**